website/docs/styling-layout.md

```diff
@@ -101,6 +101,29 @@ function MyComponent() {
 }
 ```
 
+#### Theme Class Names
+
+`@docusarus/theme-common` provides some predefined CSS class names to provide access for developers to style layout of a page globally in Docusaurus. The purpose is to have stable classnames shared by all themes that are meant to be targeted by custom CSS.
+
+```ts
+export const ThemeClassNames = {
```

Comment:
If  we refactor  `ThemeClassNames` isn't there a risk to forget to update this code snippet, and we end up with a  code snippet and actual classnames not in sync over time?

That's why I suggested using something similar to the Crowdin config here, to ensure we display in the code block the real code, so that it stays in sync automatically.

```
### Example configuration {#example-configuration}

The **Docusaurus v2 configuration file** is a good example of using versioning and multi-instance:

import CrowdinConfigV2 from '!!raw-loader!@site/../crowdin-v2.yaml';
import CodeBlock from '@theme/CodeBlock';

<CodeBlock className="language-yaml" title="test">
  {CrowdinConfigV2.split('\n')
    // remove comments
    .map((line) => !line.startsWith('#') && line)
    .filter(Boolean)
    .join('\n')}
</CodeBlock>
```

https://docusaurus.io/docs/i18n/crowdin#example-configuration

You can probably use something like this to import the file content as a string, and display it in a code block:

```
import ThemeClassNamesCode from '!!raw-loader!@theme-common/src/utils/ThemeClassNames.ts';
```

Reply:
Ah, I see. I didn't get what you meant before, going to import it from that file now.

Reply:
I have updated the documentation and imported the code by using `raw-loader` but right now I imported it like this:
`import ThemeClassNamesCode from '!!raw-loader!@site/../packages/docusaurus-theme-common/src/utils/ThemeClassNames.ts'; import CodeBlock from '@theme/CodeBlock';`

Is there a way to import it from `@theme-common`? because I tried before and it couldn't find the file.